redirects.yaml

```diff
@@ -785,6 +785,11 @@ redirects:
 - from: /docs/privacy-sandbox/permissions-policy/
   to: /articles/permissions-policy/
 
+# PR 7600 replacing Custom Tabs native apps how-to with external handler section in warmup/prefetch guide
+
+- from: /docs/android/custom-tabs/howto-custom-tab-native-apps/
+  to: /docs/android/custom-tabs/guide-warmup-prefetch/#open-webpages-in-native-apps
```

Comment:
You need the special anchor link redirect syntax here (documented in the handbook). However, this will stop working once we've moved to DevSite.

Reply:
Does that work if I'm redirecting a whole page? It looks like the anchor redirect syntax needs a subheading.

Reply:
@jpmedley what does the syntax look like without a source anchor? Latest commit has an attempt.

Reply:
This is more Sebastian's content than mine. I think the only solution, as bad as it is, is to remove the anchor from the redirect and call it done.

Reply:
+1 to Joe, this is going to stop working anyway soon.